services/parse/auth/index.js

```diff
@@ -49,6 +49,24 @@ function retrieveForgotPasswordFunction(params) {
 }
 
 function retrieveCurrentUserFunction() {
+//   ASYNC VERSION -- needed this to switch to data collection as home screen
```

Comment:
@jamccomb92 why is this commented out? can keep it as is even if we're not using it